manta-js/README.md

```diff
@@ -0,0 +1,258 @@
+
+```javascript
+// DOL token
+const assetId = new BN("1");
+const amount = new BN("10000000000000000000");
+
+// Sync with most recent ledger state.
+await privateWallet.initialWalletSync();
+
+// Get private address
+const privateAddress = await privateWallet.getPrivateAddress();
+
+// Get private balance of DOL for given private address
+const privateBalance = await privateWallet.getPrivateBalance(assetId);
+
+// Privatize 10 DOL to 10 pDOL
+await privateWallet.toPrivateSend(assetId, amount, polkadotSigner, polkadotAddress);
+
+// Sync to get latest data after the transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should be incremented by 10 units.
+const newPrivateBalance = await mantaSdk.getPrivateBalance(assetId);
+```
+
+### Private Transfer
+
+This example transfers 10 private private pDOL to another address.
+
+```javascript
+// DOL token
+const assetId = new BN("1");
+const amount = new BN("10000000000000000000");
+
+// Sync with most recent ledger state.
+await privateWallet.initialWalletSync();
+
+// Get private address
+const privateAddress = await privateWallet.getPrivateAddress();
+
+// Private Transfer of 10 pDOL to another private address
+const examplePrivateAddress = "3UG1BBvv7viqwyg1QKsMVarnSPcdiRQ1aL2vnTgwjWYX";
+await privateWallet.privateTransferSend(assetId, amount, examplePrivateAddress, polkadotSigner, polkadotAddress);
+
+// Sync to get latest data after transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should decrease by 10 units.
+const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+```
+
+### To Public
+
+This example converts 5 private pDOL to 5 public DOL.
+
+```javascript
+// DOL token
+const assetId = new BN("1");
+const amount = new BN("5000000000000000000");
+
+// Sync with most recent ledger state.
+await privateWallet.initialWalletSync();
+
+// Get private address
+const privateAddress = await privateWallet.getPrivateAddress();
+
+// Get private balance of DOL for given private address
+const privateBalance = await privateWallet.getPrivateBalance(assetId);
+
+// Convert 5 pDOL back to DOL
+await privateWallet.toPublicSend(assetId, amount, polkadotSigner, polkadotAddress);
+
+// Sync to get latest data after transaction and check that it was successful.
+await privateWallet.walletSync();
+
+// The private balance of pDOL should decrease by 5 units.
+const newPrivateBalance = await privateWallet.getPrivateBalance(assetId);
+```
+
+### Manta Utilities
+
+There also exists a `MantaUtilities` class with additional functions. Mainly for interacting publicly with the Manta ecosystem. This example demonstrates these functions. This example assumes the `MantaPrivateWallet` class has already been initialized, as well as `polkadotAddress` and `polkadotSigner`.
+
+```javascript
+import { MantaUtilities } from "manta.js";
+
+const mantaUtilities = new MantaUtilities();
```

Comment:
same thing here we can get rid of this class instance and use a static version of it